shuffle/cli/Cargo.toml

```diff
@@ -42,8 +42,10 @@ move-package = { path = "../../language/tools/move-package" }
 move-binary-format = { path = "../../language/move-binary-format" }
 shuffle-custom-node = { path = "../genesis" }
 shuffle-transaction-builder = { path = "../transaction-builder" }
-serde-reflection = "0.3.4"
-serde-generate = "0.20.2"
+serde-reflection = { git = "https://github.com/novifinancial/serde-reflection" }
+serde-generate = { git = "https://github.com/novifinancial/serde-reflection" }
+#serde-reflection = "0.3.4"
+#serde-generate = "0.20.2"
```

Comment:
mistake?

Reply:
This is currently being built off of a not-yet-released serde-reflection/generate. Before landing this will be updated to point at a valid version on crates.io.